stranger/cli.py

```diff
@@ -70,17 +86,20 @@ def cli(context, vcf, repeats_file, loglevel):
             if line.startswith('##'):
                 click.echo(line)
                 continue
-            # Print the new header line describing stranger annotation
-            click.echo(stranger_header)
+            # Print the new header lines describing stranger annotation
+            for header in stranger_headers:
+                click.echo(header)
             # Print the vcf header line
             header_info = line[1:].split('\t')
             click.echo(line)
             continue
         variant_info = dict(zip(header_info, line.split('\t')))
         variant_info['alts'] = variant_info['ALT'].split(',')
         variant_info['info_dict'] = get_info_dict(variant_info['INFO'])
-        repeat_string = get_repeat_info(variant_info, repeat_information)
-        if repeat_string:
-            variant_info['info_dict'][stranger_info] = repeat_string
+        repeat_data = get_repeat_info(variant_info, repeat_information)
+        if repeat_data:
+            variant_info['info_dict']['STR_STATUS'] = repeat_data['repeat_strings']
+            variant_info['info_dict']['STR_NORMAL_MAX'] = str(repeat_data['lower'])
+            variant_info['info_dict']['STR_FULLMUT_MIN'] = str(repeat_data['upper'])
```

Comment:
For our international contributors (and the poor guys who types in the values in the json definition file) could we keep this as PATHOLOGIC_MIN instead of FULLMUT_MIN!